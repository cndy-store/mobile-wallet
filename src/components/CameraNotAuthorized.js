import React, { Component } from 'react';
import { View, Text } from 'react-native';

export default class CameraNotAuthorized extends Component {
  render() {
    return (
      <View
        style={{
          flex: 1,
          alignItems: 'center',
          justifyContent: 'center',
          backgroundColor: '#ff0000'
        }}
      >
        <Text
          style={{
            textAlign: 'center',
            fontSize: 16
          }}
        >
          Camera not authorized!
        </Text>
      </View>
    );
  }
}
