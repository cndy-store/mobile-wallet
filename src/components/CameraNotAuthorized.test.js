import React from 'react';
import CameraNotAuthorized from './CameraNotAuthorized.js';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<CameraNotAuthorized />).toJSON();
  expect(tree).toMatchSnapshot();
});
