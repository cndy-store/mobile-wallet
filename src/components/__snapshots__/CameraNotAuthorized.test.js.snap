// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ff0000",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 16,
        "textAlign": "center",
      }
    }
  >
    Camera not authorized!
  </Text>
</View>
`;
